build.sbt

```diff
@@ -22,7 +22,7 @@ lazy val coverageSettings: Seq[Setting[?]] = {
 
   Seq(
     ScoverageKeys.coverageExcludedPackages := excludedPackages.mkString(";"),
-    ScoverageKeys.coverageMinimumStmtTotal := 78,
+    ScoverageKeys.coverageMinimumStmtTotal := 70,
```

Comment:
why are the decreasing the code coverage. This should be 80

Reply:
Reverted